packages/botonic-dx/README.md

```diff
@@ -18,7 +18,7 @@ npm install -D @botonic/dx
 ```
 * Copy the contents of the `sample-config` folder to the root of your project 
   (Merge this package.json's scripts into your project ones)
-* These files just import the configuration maintained within this project. 
+* The files at `sample-config` just import the configuration maintained within this project. 
```

Comment:
I don't understand this sentence, what are you trying to say here? Are any files in `sample-config` that are not maintained within this project?

Reply:
What I mean is that we want to avoid copying from a template config file to the host project, because then maintaining them would require modifying each project which uses botonic-dx.
So the only files which are copied (the sample-config) are just stubs (imports) to the baseline config

Reply:
Thanks for the explanation. 
I read it in a wrong way before and after reading it again it make sense.